### ISSUE TITLE
INTERFACING LOAD CELL WITH STM 32 F4

### ISSUE THREAD

**Issue by mikeshams** [ankitpatel9300/Capstone-Project--Summer-2020#7]

**Comment by mikeshams** [comment]
Demonstrate how STM32F4 instead of LPC 1769 can resolve the issue connecting the load cell to the MCU

**Comment by ankitpatel9300** [comment]
Depending on MCU frequency, Maximum GPIO pin switching is different.
We tried with LPC1769 but can not read the proper value from the HX711 IC(i.e LoadCell).
When we tried it with STM32F4 MCU, with similar code it worked. The possible reason 
for this could be GPIO switching speed due to different frequencies. 
To correctly measure timings, we need a logic analyzer, which is not possible due to the current situation.

That's why we tried to interface Load Cell with STM32F4 and is working correctly.
